_posts/2015-03-31-how-we-do-css.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: post
+title: How We Do CSS at Zumba
+description: This is how we do CSS at Zumba.
+tags: [technology, css, sass, front end, engineering]
+author: neptunz
+---
+
+As you may or may not know Zumba's website consists of a robust app that includes a fully custom ecommerce shop that supports international transactions, class and training search and registration, Zumba Instructor Network (ZIN) admin portal and many other apps. Our in-house tech department is responsible for managing the interactive products, developing, testing and deploying the projects.
```

Comment:
Should be "Our in-house tech department is responsible for managing the interactive products, <em>as well as</em> developing, testing and deploying the projects.